lib/unbrick.js

```diff
@@ -0,0 +1,27 @@
+var update = require('./update-fetch');
+var logs = require('./logs');
+var flash = require('./flash');
+
+module.exports = function unbrick(opts, tessel) {
+  return new Promise(function(resolve, reject) {
+    if (opts.openwrt) {
+      logs.info('Proceeding with updating OpenWrt...');
+
+      // download images
+      return update.fetchUnbrick()
+        .then(function(result) {
+          flash(tessel.connection, result.uboot, result.squashfs, function onFlash(err) {
+            if (err) {
+              return reject(err);
+            }
+            resolve();
+          });
+        })
+        .catch((e) => {
```

Comment:
Are you sure this `catch` is needed if you are `return`ing the promise returned by `fetchUnbrick`?